Workbooks/SapMonitor/SapHana/SapHana.workbook

```diff
@@ -21,19 +21,115 @@
             "type": 1,
             "value": "1",
             "isHiddenWhenLocked": true
+          },
+          {
+            "id": "731f3dea-e655-4c54-8db1-79e6d4691709",
+            "version": "KqlParameterItem/1.0",
+            "name": "param_check_error",
+            "type": 1,
+            "query": "// 0 = no error\n// 1 = table does not exist (need to add provider)\n// 2 = table exists, but column v1 does not (need to wait)\n// 3 = table exists, column v1 too, but not column v2 (need to upgrade)\n// 4 = table exists, column v1 and column v2 as well, but no records (need to wait)\nlet t = \"SapHana_LoadHistory_CL\";\nlet c1 = \"HOST_s\";\nlet c2 = \"PROVIDER_INSTANCE_s\";\nlet dummy = \"unmöglicher-wert!\";\nlet MissingTable = view () { print tableMissing = 1, columnV1 = dummy, columnV2 = dummy, records = -1 };\nlet checkRecords =\n  union\n    isfuzzy=true MissingTable,\n    (table(t)\n      | summarize records = count()\n    )\n  | project records\n  | top 1 by records desc\n;\nlet checkExists =\n  union\n    isfuzzy=true MissingTable,\n    (table(t)\n      | extend tableMissing = 0\n      | extend columnV1 = column_ifexists(c1, dummy)\n      | extend columnV2 = column_ifexists(c2, dummy)\n    )\n  | project tableMissing, columnV1, columnV2\n  | limit 2\n  | top 1 by tableMissing asc\n;\nlet tableExists = toscalar(checkExists | summarize max(tableMissing) != 1);\nlet columnV1Exists = toscalar(checkExists | summarize max(columnV1) != dummy);\nlet columnV2Exists = toscalar(checkExists | summarize max(columnV2) != dummy);\nlet tableRecords = toscalar(checkRecords | summarize max(records));\nlet x = iif(tableExists, iif(columnV1Exists, iif(columnV2Exists, iif(tableRecords > 0, \"0\", \"2\"), \"3\"), \"2\"), \"1\");\nprint toscalar(x)\n//print tableExists, columnV1Exists, columnV2Exists, tableRecords\n",
```

Comment:
dummy = \"unmöglicher-wert!\" ??

Reply:
Some hardcore Kusto magic going on here...
Basically setting dummy to an "impossible value" (that's what that means).